packages/sdk-core/CHANGELOG.md

```diff
@@ -12,7 +12,12 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - Added `PageNumberPaging` for UI development friendly pagination API
 - Added `AllPaging` to recursively query all results from Subgraph
 - Added support for `TypedDocumentNode` for `SubgraphClient`
+- Expose underlying ethers.js contracts for each class: CFAv1, Host, IDAv1 and `contracts` property in `Framework` class
 - Added new ACL function support: authorizing flow operator permissions and create/update/delete flow by operator
+- Added `nativeTokenSymbol` property to `constants.ts`
+- Split `SuperToken` class into: `SuperTokenWithUnderlying`, `SuperTokenWithoutUnderlying` and `NativeAssetSuperToken` classes
```

Comment:
"SuperTokenWithUnderlying" & "SuperTokenWithoutUnderlying"

Don't we have better terminology for this?

@hellwolf

Reply:
okay, largely reasonable, though NativeAssetSuperToken and SuperTokenWithoutUnderlying technically can overlap.

Reply:
in protocol, it is `createERC20Wrapper` vs `initializeCustomSuperToken`

as a reference, the current classification system is documented in https://www.notion.so/superfluidhq/Classification-of-Super-Tokens-5beace780b5c4d09a5752a3677da3dc0

Reply:
Is it possible that I've heard the term "native super token" for super tokens that don't have an underlying? Because that's the association I had in my head before this PR.

Reply:
okay, naming discussed and addressed